mmv1/third_party/cai2hcl/common/hcl_write.go

```diff
@@ -3,10 +3,25 @@ package common
 import (
 	"fmt"
 
+	"github.com/hashicorp/hcl/hcl/printer"
 	"github.com/hashicorp/hcl/v2/hclwrite"
 	"github.com/zclconf/go-cty/cty"
 )
 
+func HclWriteBlocks(blocks []*HCLResourceBlock) ([]byte, error) {
```

Comment:
Splitting this out seems like a good idea! 👍 I sort of wish we had a name other than HCLResourceBlock for the input since it's not actually an HCL-specific format, just an intermediate container... but that's not something to change right now.

I'd be fine to approve and merge this change in a separate PR if you want but it's not really impacting readability.